integration/src/main.rs

```diff
@@ -263,6 +267,50 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     Ok(())
 }
 
+fn load_wasm(wasm_bytes: &[u8]) -> (Module, Vec<WriteOnceAsm<Directive<F>>>) {
+    let PartiallyParsedProgram {
+        s: settings,
+        m: mut module,
+        functions,
+    } = womir::loader::load_wasm(OpenVMSettings::<F>::new(), wasm_bytes).unwrap();
+
+    // Compile the functions.
+    let mut label_gen = 0..;
+    let mut stats = Statistics::default();
+    let mut tracker = builtin_functions::Tracker::new();
+    let mut functions = functions
+        .into_iter()
+        .enumerate()
+        .map(|(idx, mut func)| {
+            let idx = idx as u32;
+
+            // Advance to BlocklessDag stage
+            let dag = loop {
+                if let FunctionProcessingStage::BlocklessDag(dag) = &mut func {
+                    break dag;
+                }
+                func = func
+                    .advance_stage(&settings, &module, idx, &mut label_gen, Some(&mut stats))
+                    .unwrap();
+            };
+
+            // In the BlocklessDag stage, we need to find instructions we don't implement and replace
+            // them with function calls to built-in functions.
+            tracker.replace_with_builtins(&mut module, &mut label_gen, dag);
+
+            // Advance to final stage
+            func.advance_all_stages(&settings, &module, idx, &mut label_gen, Some(&mut stats))
+                .unwrap()
+        })
+        .collect_vec();
+
+    functions.extend(tracker.into_used_builtins());
+
+    println!("WOMIR loading statistics: {stats}");
```

Comment:
log info?

Reply:
this would be a bigger change, because there are other prints

Reply:
ah, there is not even log crate as dependency